lib/results_validators/persons_validator.rb

```diff
@@ -41,22 +41,22 @@ def include_persons?
       true
     end
 
-    def self.dob_validations(dob, competition_id = nil, name = nil)
+    def self.dob_validations(dob, message_args = {}, competition_id = nil)
```

Comment:
I think you can make it more Ruby-esque by something like...
```suggestion
    def self.dob_validations(dob, competition_id = nil, **message_args)
```

Reply:
If you take this suggestion, then below you can "spread" the parameters into the `ValidationIssue` class as well: `ValidationWarning.new(:persons, competition_id, DOB_0101_WARNING, **message_args)`

Reply:
Okay done.